doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -440,6 +444,10 @@ Wi-Fi samples
 
   * :ref:`wifi_sr_coex_sample` sample demonstrating Wi-Fi Bluetooth LE coexistence.
 
+* Removed:
+
+  * nRF7002 revision A support.
+
```

Comment:
Can this be added only under one section instead of Protocols and Samples both?

Reply:
Ok, but which one?

Reply:
Actually IMHO it might be better to highlight in both, as we made changes in both areas.